app/src/main/java/com/github/polypoly/app/MapActivity.kt

```diff
@@ -154,6 +161,48 @@ class MapActivity : ComponentActivity() {
         }
     }
 
+    @Composable
+    fun Hud(data: PlayerGlobalData, round: Int) {
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Box(
+                modifier = Modifier
+                    .padding(16.dp)
+                .background(Color.White)
+                .border(1.dp, Color.Black)
+                .align(Alignment.TopStart)
+            ) {
+                Row(modifier = Modifier.padding(8.dp)) {
+                    Button(
+                        onClick = {},
+                        modifier = Modifier
+                            .size(70.dp)
+                            .semantics { contentDescription = "" },
+                        shape = RoundedCornerShape(10.dp),
```

Comment:
In the graphic chart we use round buttons when they are small. But it's important to ask yourself what this button is for, because it might be more relevant to make it look different from the graphic chart and to put it bigger, and here with your code I'm not sure I understand what it is for.

Reply:
It's a button that displays the full game information, so I think it should be the same size as the buttons, what do you think? I've made this clearer in the code.